packages/babel-plugin-transform-spread/src/index.js

```diff
@@ -94,7 +94,14 @@ export default declare((api, options) => {
         const args = node.arguments;
         if (!hasSpread(args)) return;
 
-        const calleePath = path.get("callee");
+        let calleePath = path.get("callee");
+
+        if (calleePath.isTSAsExpression()) {
+          do {
+            calleePath = calleePath.get("expression");
+          } while (calleePath.isTSAsExpression());
+        }
```

Comment:
Using `if` before a `do-while` is exactly like using a `while`:
```suggestion
        while (calleePath.isTSAsExpression()) {
          calleePath = calleePath.get("expression");
        }
```